client/my-sites/plugins/plans/index.tsx

```diff
@@ -59,8 +61,34 @@ const Plans = ( { intervalType }: { intervalType: 'yearly' | 'monthly' } ) => {
 		);
 	}, [ dispatch, translate, selectedSite, breadcrumbs.length, intervalType ] );
 
+	const promos: PromoSectionProps = {
+		promos: [
+			{
+				title: translate( 'Flex your site with plugins' ),
+				body: translate(
+					'Install plugins and extend functionality for your site with access to more than 50,000 plugins.'
+				),
+				image: <Gridicon icon="plugins" />,
+			},
+			{
+				title: translate( 'Money back guarantee' ),
+				body: translate(
+					'Try WordPress.com for 14 days and if you are not 100% satisfied, get your money back.'
+				),
+				image: <Gridicon icon="money" />,
+			},
+			{
+				title: translate( 'Essential features' ),
+				body: translate(
+					"We guarantee site's performance and protect it from spammers detailing all activity records."
+				),
+				image: <Gridicon icon="plans" />,
+			},
+		],
+	};
+
 	return (
-		<MainComponent wideLayout>
+		<MainComponent className="plugin-plans" wideLayout>
```

Comment:
Added this main class to make sure we'll have the right scope

Reply:
It looks like we also assign this `plugin-plans` class name to the ActionCard. How do you feel about renaming it to something like `plugins-plans-main` so it's more specific to the MainComponent?

Reply:
Sounds good @DustyReagan, I also improved the CSS a little bit, so now this information is aligned with the plans columns.